x/evm/artela/contract/service.go

```diff
@@ -194,3 +243,17 @@ func (service *AspectService) GetAspectProof(ctx sdk.Context, aspectId common.Ad
 func (service *AspectService) GetBlockHeight() int64 {
 	return service.getHeight()
 }
+
+func (service *AspectService) GetAspectJoinPoint(ctx sdk.Context, aspectId common.Address, version *uint256.Int, commit bool) *big.Int {
+	if commit {
+		sdkCtx, getErr := service.getCtxByHeight(ctx.BlockHeight()-1, true)
```

Comment:
I remembered that we have discussed that we will not use block - 1 anymore. You already have this sdkCtx as your first input param, why using getCtxByHeight again? This code makes no sense.

Reply:
this issue will be resolved in a separate branch